pkg/cmd/service/add.go

```diff
@@ -55,6 +73,40 @@ func (o *AddServiceOptions) Run() error {
 	return nil
 }
 
+func checkServiceDependencies(io *AddServiceOptions, client *utility.Client, spinner status) error {
+	spinner.Start("Checking if Sealed Secrets is installed with default configuration", false)
```

Comment:
I think if the user intentionally specified these flags, then they'd better get it right, and kam should just fail with an error.

Reply:
As discussed, the user has to specify --insecure to generate unsealed secrets.   Without it, it will be sealed, and if these are values are invalid, then it will fail.